### PR TITLE
[FR] Update `Featured Artists`, add `FA Showcase Beatmaps`, and fix links

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal for the French translation of the wiki page `Featured Artists Showcase Beatmaps`.

Proposal to update the French translation of the wiki page `Featured Artists`.

Also:

- In `Featured_Artists/Featured_Artist_Showcase_Beatmaps`: Fix invalid link

### CODE REVIEW

**Comment by AncuL001** [comment]
i don't know why HyuN's and Se-U-Ra's dates need to be changed into such. it should be visible from the date of the news link right?

**Comment by Nivalyx** [comment]
yeah i agree unless it's way off dates shouldn't matter that much especially because on most cases the osu! website displays dates based on what time zone the user is currently in

image

**Comment by DoryanR** [comment]
Normally I paid attention to this detail, I changed the language on the site to avoid errors. But normally the time difference between French and English is 1h30.

**Comment by Nivalyx** [comment]
o i think you get it wrong; the dates that you're seeing are based on *the time zone you're currently in*, not what language/locale you're currently browsing the website in.

fwiw this article was authored by me (https://github.com/ppy/osu-wiki/pull/5245) who lives in gmt +7 so yeah certain differences caused by time zone lags are just imminent lol

**Comment by DoryanR** [comment]
I put the dates back the way they were.

File: wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/fr.md [hunk 1]
```diff
@@ -0,0 +1,393 @@
+# Vitrine des beatmaps des artistes associés
+
+Les membres de la [Mappers' Guild](/wiki/Mappers_Guild) créent parfois des "beatmaps vitrines" exclusives en utilisant des chansons et d'autres éléments des [artistes associés](/wiki/Featured_Artists) inédits d'osu!. Lorsqu'un artiste est annoncé publiquement, ces beatmaps sont publiées dans les [newsposts](https://osu.ppy.sh/home/news) et font souvent l'objet d'une promotion sur le [Twitter @osugame](https://twitter.com/osugame).
+
+Pour participer aux beatmaps de la vitrine des artistes associés, un membre de la Mappers' Guild doit accepter une [quête mystérieuse à durée limitée](/wiki/Mappers_Guild#quêtes) ou être directement invité à créer une beatmap après avoir atteint le niveau 1 (ou supérieur) du [rang de la guilde](/wiki/Mappers_Guild#récompenses).
+
+Cet article répertorie tous les beatmaps vitrines qui ont été créés par les membres de la Mappers' Guild au fil du temps.
```
Comment (Ganondorf-1) [inline wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/fr.md:7]:
```suggestion
Cet article répertorie toutes les beatmaps vitrines qui ont été créées par les membres de la Mappers' Guild au fil du temps.
```

File: wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/fr.md [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+  - (![][osu!]) [Aiobahn - Koko ni Iru (feat. rionos)](https://osu.ppy.sh/beatmapsets/1129013) organisé par ![][flag_MX] [Gero](https://osu.ppy.sh/users/1467715)
+  - (![][osu!taiko]) [Aiobahn - Sugiyuku Hi to Kimi e (feat. nayuta)](https://osu.ppy.sh/beatmapsets/1124134) organisé par ![][flag_HK] [Faputa](https://osu.ppy.sh/users/845733)
+  - (![][osu!mania]) [Aiobahn - Sugiyuku Hi to Kimi e (feat. nayuta)](https://osu.ppy.sh/beatmapsets/1126358) organisé par ![][flag_DE] [Feerum](https://osu.ppy.sh/users/4815717)
+- **[yukitani](https://osu.ppy.sh/home/news/2021-05-22-new-featured-artist-yukitani)** (22/05/2021)
+  - (![][osu!taiko]) [yukitani - Amusing Reflection Rag.](https://osu.ppy.sh/beatmapsets/1436446) organisé par ![][flag_AR] [Reficul](https://osu.ppy.sh/users/1506011)
+
+## Histoire
+
+Dès sa création, la Mappers' Guild s'est fixé comme objectif principal de "créer des packs de beatmaps promotionnels pour les artistes associés à venir". En réalité, les six beatmaps réalisées pour l'[annonce de l'artiste associés MOtOLOiD](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid) étaient également les toutes premières créations de la Guilde.
```
Comment (Ganondorf-1) [inline wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/fr.md:354]:
```suggestion
Dès sa création, la Mappers' Guild s'est fixé comme objectif principal de "créer des packs de beatmaps promotionnels pour les artistes associés à venir". En réalité, les six beatmaps réalisées pour l'[annonce de l'artiste associé MOtOLOiD](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid) étaient également les toutes premières créations de la Guilde.
```

File: wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/fr.md [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+
+Comme la Guilde elle-même fonctionnait à ses débuts sur la base d'une inscription fermée, l'accès aux contenus des artistes associéss à venir était souvent limité à une poignée de mappeurs qui étaient principalement du mode de jeu [osu!](/wiki/Game_mode/osu!); cependant, le [passage de la Guilde à un système d'inscription ouvert et automatisé au début de 2019](/wiki/Mapping_and_Modding_Timeline#march.5 ) a progressivement permis à de plus en plus de créateurs de participer à la création de beatmaps vitrines pour les futurs artistes associés au fil du temps, quel que soit leur mode de jeu.
```
Comment (Ganondorf-1) [inline wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/fr.md:356]:
```suggestion
Comme la Guilde elle-même fonctionnait à ses débuts sur la base d'une inscription fermée, l'accès aux contenus des artistes associés à venir était souvent limité à une poignée de mappeurs qui étaient principalement du mode de jeu [osu!](/wiki/Game_mode/osu!); cependant, le [passage de la Guilde à un système d'inscription ouvert et automatisé au début de 2019](/wiki/Mapping_and_Modding_Timeline#march.5 ) a progressivement permis à de plus en plus de créateurs de participer à la création de beatmaps vitrines pour les futurs artistes associés au fil du temps, quel que soit leur mode de jeu.
```

File: wiki/Featured_Artists/fr.md [hunk 4]
```diff
@@ -1,16 +1,41 @@
...
 
-## Mappers' Guild
+## Sélection
 
-Pour promouvoir le contenu des artistes associés, les membres de la communauté ont créé la **Mappers' Guild**, offrant aux mappeurs un système de quêtes avec un style MMO, auquel ils peuvent participer avec d'autres mappeurs. Les récompenses pour les membres les plus actifs de la guilde incluent des badges de profil, un accès rapide aux nouveaux contenus d’artistes associés et la possibilité d’aider à décider avec quels artistes l'équipe d'osu! devrait envisager de travailler dans le futur.
+Les ajouts récents à la bibliothèque des artistes associés sont sélectionnés indirectement par la communauté d'osu!. Si des mappers créent et classent des maps pour un artiste, l'[équipe d'osu!](/wiki/People/The_Team) contactent l'artiste pour discuter du programme d'artistes associés. Cependant, certains artistes sont parfois invités à devenir des artistes associés en fonction d'autres facteurs, tels que l'intérêt qu'ils devraient présenter pour le public du beatmapping d'osu!
```
Comment (Ganondorf-1) [inline wiki/Featured_Artists/fr.md:14]:
```suggestion
Les ajouts récents à la bibliothèque des artistes associés sont sélectionnés indirectement par la communauté d'osu!. Si des mappeurs créent et classent des maps pour un artiste, l'[équipe d'osu!](/wiki/People/The_Team) contacte l'artiste pour discuter du programme d'artistes associés. Cependant, certains artistes sont parfois invités à devenir des artistes associés en fonction d'autres facteurs, tels que l'intérêt qu'ils devraient présenter pour le public du beatmapping d'osu!
```

File: wiki/Featured_Artists/fr.md [hunk 5]
```diff
@@ -1,16 +1,41 @@
...
 
-Plus d'informations sur la Mappers' Guild sont disponibles sur [leur site internet](http://mappersguild.com) et dans le canal `#mappers-guild` du [serveur Discord osu!dev](https://discord.gg/ppy).
+La plupart des chansons figurant dans le programme des artistes associés font l'objet d'une licence non exclusive pour osu!. Dans de rares cas, un artiste créera des morceaux exclusivement pour osu!, qui seront étiquetté *osu! exclusive* sur leur liste d'artistes associés.
```
Comment (Ganondorf-1) [inline wiki/Featured_Artists/fr.md:16]:
```suggestion
La plupart des chansons figurant dans le programme des artistes associés font l'objet d'une licence non exclusive pour osu!. Dans de rares cas, un artiste créera des morceaux exclusivement pour osu!, qui seront étiquetés *osu! exclusive* sur leur liste d'artistes associés.
```

File: wiki/Featured_Artists/fr.md [hunk 6]
```diff
@@ -1,16 +1,41 @@
...
+Les nouvelles sorties d'artistes associés par an :
+
+- **2016 :** 8
+- **2017 :** 11
+- **2018 :** 20
+- **2019 :** 31
+- **2020 :** 65
+- **2021 :** *à déterminer*
+
+Les nouveaux artistes associés sont annoncés chaque semaine sur le [fil d'actualités](https://osu.ppy.sh/home/news) d'osu!. La plupart des annonces comprennent une liste des chansons sous licence de l'artiste et un lien vers sa page d'artiste associés, mais certaines annonces sont associées à des concours de mapping ou à des beatmaps créés par la [Guilde des mappeurs](/wiki/Mappers_Guild).
```
Comment (Ganondorf-1) [inline wiki/Featured_Artists/fr.md:31]:
```suggestion
Les nouveaux artistes associés sont annoncés chaque semaine sur le [fil d'actualités](https://osu.ppy.sh/home/news) d'osu!. La plupart des annonces comprennent une liste des chansons sous licence de l'artiste et un lien vers sa page d'artiste associé, mais certaines annonces sont associées à des concours de mapping ou à des beatmaps créés par la [Mappers' Guild](/wiki/Mappers_Guild).
```

File: wiki/Featured_Artists/fr.md [hunk 7]
```diff
@@ -1,16 +1,41 @@
...
+
+## Utilisation par la communauté
+
+Les mappers sont encouragés à créer des maps pour les pistes des artistes associés. Ces morceaux sont fournis avec des données de [timing](/wiki/Beatmap_Editor/Timing) et sont déjà autorisés à être utilisés dans osu!, ce qui permet aux utilisateurs de se lancer immédiatement dans la création de beatmap sans crainte. La version en cours de développement d'osu!, appelée [osu!lazer](https://github.com/ppy/osu/releases), mettra ces maps en évidence dans le jeu.
```
Comment (Ganondorf-1) [inline wiki/Featured_Artists/fr.md:35]:
```suggestion
Les mappeurs sont encouragés à créer des maps pour les pistes des artistes associés. Ces morceaux sont fournis avec des données de [timing](/wiki/Beatmap_Editor/Timing) et sont déjà autorisés à être utilisés sur osu!, ce qui permet aux utilisateurs de se lancer immédiatement dans la création de beatmap sans crainte. La version en cours de développement d'osu!, appelée [osu!lazer](https://github.com/ppy/osu/releases), mettra ces maps en évidence dans le jeu.
```

File: wiki/Featured_Artists/fr.md [hunk 8]
```diff
@@ -1,16 +1,41 @@
...
+
+Tous les concours de mapping publiés sur la [liste des concours communautaires](https://osu.ppy.sh/community/contests), tels que le [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest) et l'[Mapping Olympiad](/wiki/Contests#osu!-mapping-olympiad), exigent des participants qu'ils fassent les chansons des artistes associés. Certains concours organisés par la communauté utilisent également ces chansons.
```
Comment (Ganondorf-1) [inline wiki/Featured_Artists/fr.md:37]:
```suggestion
Tous les concours de mapping publiés sur la [liste des concours communautaires](https://osu.ppy.sh/community/contests), tels que le [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest) et le [Mapping Olympiad](/wiki/Contests#osu!-mapping-olympiad), exigent des participants qu'ils mappent les chansons des artistes associés. Certains concours organisés par la communauté utilisent également ces chansons.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by Ganondorf-1 — APPROVED** [review]

**Review by DoryanR — COMMENTED** [review]
Can be merge 👍

**Review by TicClick — APPROVED** [review]

**Pull request merged**